### DISCUSSION TITLE
What slider do you recommend for lightGallery?

### DISCUSSION THREAD

**Discussion by addison74** [sachinchoolur/lightGallery#1249] [Q&A]
Years in a row I used PhotoSwipe but when I saw that there are tons of unresolved issues that no new versions appear, I looked for alternatives. I recently evaluated fancyBox and lightGallery and came to the conclusion that the latter one is far more elaborate, the number of issues is low, PR's are analyzed and integrated quickly.

I would like to create a thumbnails slider and when an image is clicked it will be opened with lightGallery. I analyzed the existing solutions based on the same criteria and I've made a list Swiper, Splide, Glide, Glider, Flickity. The first two attracted me from the perspective of documentation, features and intuitive implementation. I would appreciate if those who have experience with these JS scripts would say an opinion. Which would be the best solution in combination with lightGallery?

Please note that I also evaluated lightSlider but the project appears abandoned, in addition it depends on JQuery.

**Comment by sachinchoolur** [comment]
Hey @ADDISON74,

I had used Swiper earlier, and it is really nice. Let me know if you need any help integrating lightGallery with Swiper or any other slider.

**Reply by webdiva69** [reply]
@sachinchoolur do you have any instructions to integrate lightGallery with Swiper? We already have a swiper carousel and we want to be able to pop open a light box of the same images.

**Reply by sachinchoolur** [reply]
I created a demo of using lightGallery with SwiperJs - https://www.lightgalleryjs.com/demos/swiper/

let me know if you face any issues
